### DISCUSSION TITLE
Disable copyQ when specific window is focussed (e.g. password manager)

### DISCUSSION THREAD

**Discussion by nkempe** [hluk/CopyQ#1690] [Ideas]
CopyQ also remembers text copied from my password manager.
This is a bit of a security risk.

Can CopyQ be configured so that it automatically disables as long as a specific application has focus?
It would be a great feature to improve general security.

FYI: I'm using KeepassXC right now, but that might change.

**Comment by hluk** [comment]
See: https://copyq.readthedocs.io/en/latest/faq.html#how-to-omit-storing-text-copied-from-specific-windows-like-a-password-manager

Specifically for **KeePassXC**: https://github.com/hluk/copyq-commands/tree/master/Automatic#keepassxc-protector (see [README.md](https://github.com/hluk/copyq-commands/blob/master/README.md) for details on how to add the command)

**Reply by nkempe** [reply]
That's perfect, thanks!
Sorry I missed the docs..
